backend/danswer/server/manage/users.py

```diff
@@ -159,12 +160,18 @@ def bulk_invite_users(
     emails: list[str] = Body(..., embed=True),
     current_user: User | None = Depends(current_admin_user),
 ) -> int:
+    """emails are string validated. If any email fails validation, no emails are
+    invited and an exception is raised."""
     if current_user is None:
         raise HTTPException(
             status_code=400, detail="Auth is disabled, cannot invite users"
         )
 
-    all_emails = list(set(emails) | set(get_invited_users()))
+    normalized_emails = []
+    for email in emails:
+        email_info = validate_email(email)  # can raise EmailNotValidError
+        normalized_emails.append(email_info.normalized)  # type: ignore
+    all_emails = list(set(normalized_emails) | set(get_invited_users()))
```

Comment:
any potential issue with having duplicates (like a normalized and a non normalized version) in the DB?

Reply:
Before the code would have allowed anything into the database, so this is a strict improvement.

I don't see any issues with this on the read end of things.  At worst the email is in the database in more than one form, but we'll still only pass emails that match.

Moving forward we could add some code to renormalize existing emails, but I think we should get this change out first.